packages/ssi-service/src/default.html

```diff
@@ -68,12 +71,15 @@
         }
         styleTag.remove();
         document.body.prepend( opfNav );
-        document.body.append( opfFeedback );
       } );
     }
 
     const opfNav = document.createElement( "op-nav" );
-    const opfFeedback = document.createElement( "op-feedback" );
+    document.write('<opc-feedback></opc-feedback>');
+    document.querySelector('opc-feedback').addEventListener('opc-feedback:submit', (event) => {
+      event.detail.data.createdBy = window.OpAuthHelper.getUserInfo().rhatUUID;
+      window.sendFeedback( event.detail.data);
+    } );
```

Comment:
Do not use document.write for adding html elements in the DOM. You can create the element using document.createElement and add it to the DOM using append or prepend.
```suggestion
    const opcFeedback = document.createElement('opc-feedback');
    opFeedback.on('opc-feedback:submit', 
    opFeedback.on('opc-feedback:submit', (event) => {
      event.detail.data.createdBy = window.OpAuthHelper?.getUserInfo()?.rhatUUID;
      window.sendFeedback( event.detail.data);
    } );
```